fw/access_log.c

```diff
@@ -413,21 +539,91 @@ do_access_log(TfwHttpResp *resp)
 			  TFW_HTTP_RESP_CUT_BODY_SZ(resp));
 }
 
+static int
+cfg_access_log_set(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	int i;
+	const char *val;
+	bool off = false;
+
+	TFW_CFG_CHECK_VAL_N(>, 0, cs, ce);
+	TFW_CFG_CHECK_NO_ATTRS(cs, ce);
```

Comment:
Here we have a problem:
```
access_log mmap mmap_log="/tmp/1" mmap_host=localhost;
access_log dmesg;
```
Not work because mmap has two attributes (and it is true, because each value=key is attribute).
If we not add `access_log dmesg;` all works fine, but it seems that it is because of our parser fails when attribute name starts from the same as characters as value.

Reply:
Fixed incorrect options removing and made the script to handle all access_log tags.